playbooks/roles/caasp4/tasks/setup_nodes.yml

```diff
@@ -56,6 +56,19 @@
     src: inventory-caasp4.yml.j2
     dest: "{{ socok8s_workspace }}/inventory/caasp4.yml"
 
+- name: Extend inventory to co-use the LB node as OSH deployer node
```

Comment:
how would this work if the ip is not really a node? Any further tasks that want to use the deployer node for a play will fail.

Reply:
In case of a kvm/libvirt deployment it is always real node. I just forgot to skip for openstack deployments.